packages/sprintf/README.md

```diff
@@ -38,6 +38,23 @@ sprintf( 'Hello %s!', 'world' );
 // ⇒ 'Hello world!'
 ```
 
+## Type coercions and default values
+
+When replacing numeric types (`%d` and `%f`), values will be coerced to numeric values, and default to 0:
+- `sprintf( '%d', 123 )` returns `'123'`
+- `sprintf( '%d', '123' )` returns `'123'`
+- `sprintf( '%d', 'string' )` returns `'0'`
```

Comment:
These examples read like good test cases, yet I don't think we have coverage for some, like this one?

Reply:
Good point! I added the remaining ones.